### DISCUSSION TITLE
Error : superset_app : worker(pid) was sent SIGKILL! perhaps out of memory

### DISCUSSION THREAD

**Discussion by Achintyarai22** [apache/superset#31299] [Q&A / Help]
I’m running Apache Superset, and I’ve noticed the following error in the logs:

Error : superset_app : worker(pid) was sent SIGKILL! perhaps out of memory

**Comment by jose-vines-toteat** [comment]
I too am seeing that repeatedly in my logs, with no apparent reason why 🤔

**Comment by jose-vines-toteat** [comment]
@Achintyarai22 figured out I had a bad revision provisioning machines on Google Cloud Run. Maybe check if that's the case for you.

**Comment by LeandroMartinMacato** [comment]
Hello I am also encountering this

**Comment by jokester** [comment]
The "perhaps out of memory" is but a guess of the cause. Likely hardcoded in web server for SIGKILL.

In my own case which guided me to this thread: I had a evaluation config running under guvicorn and with no redis or celery . Mine repeatedly prints the following, and the cause is likely a timeout, rather than OOM:

> [2026-06-15 14:24:03 +0900] [249632] [CRITICAL] WORKER TIMEOUT (pid:3316443)
> [2026-06-15 14:24:04 +0900] [249632] [ERROR] Worker (pid:3316443) was sent SIGKILL! Perhaps out of memory?
> [2026-06-15 14:24:04 +0900] [3328562] [INFO] Booting worker with pid: 3328562
